doc/classes/TextEdit.xml

```diff
@@ -1154,6 +1154,9 @@
 		<member name="editable" type="bool" setter="set_editable" getter="is_editable" default="true" keywords="readonly, disabled, enabled">
 			If [code]false[/code], existing text cannot be modified and new text cannot be added.
 		</member>
+		<member name="empty_selection_clipboard_enabled" type="bool" setter="set_empty_selection_clipboard_enabled" getter="is_empty_selection_clipboard_enabled" default="true">
+			If [code]true[/code], when copying or cutting without a selection, the corresponding action is performed for the line where the caret is located.
```

Comment:
```suggestion
			If [code]true[/code], copying or cutting without a selection is performed on all lines with a caret. Otherwise, copy and cut require a selection.
```
Maybe something like this would be more clear, and this works with multiple carets.
Same with the other description.